csm_web/frontend/src/components/Policies.tsx

```diff
@@ -0,0 +1,498 @@
+            <p>
+              {forUnit.body.p5_1}
+              <a href={forUnit.body.p5_a.link}>{forUnit.body.p5_a.text}</a>
+              {forUnit.body.p5_2}
+            </p>
+            <p> {forUnit.body.p6}</p>
+            <ul>
+              <li> {forUnit.body.li2_1} </li>
+              <li>
+                {forUnit.body.li2_2.bullet}
+                <ul>
+                  <li>
+                    {forUnit.body.li2_2.subbull_1}
+                    <b>{forUnit.body.li2_2.subbull_b}</b>
+                    {forUnit.body.li2_2.subbull_2}
+                  </li>
+                </ul>
+              </li>
+              <li>
+                {forUnit.body.li2_3.bullet}
+                <ul>
+                  <li>
+                    <b>{forUnit.body.li2_3.subbull_b}</b>
+                    {forUnit.body.li2_3.subbull_2}
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </div>
+
+          <div className="divider"></div>
+
+          <div>
+            <h1> {faq.title} </h1>
+            <p>
+              {" "}
+              <b> {faq.body.p1_b} </b>{" "}
+            </p>
+            <p> {faq.body.p2} </p>
+            <p>
+              {" "}
+              <b> {faq.body.p3_b} </b>{" "}
+            </p>
+            <p> {faq.body.p4} </p>
+            <p>
+              {" "}
+              <b> {faq.body.p5_b} </b>{" "}
+            </p>
+            <p> {faq.body.p6} </p>
+            <p>
+              {" "}
+              <b> {faq.body.p7_b} </b>{" "}
+            </p>
+            <p>
+              {faq.body.p8}
+              <ul>
+                <li> {faq.body.li1} </li>
+                <li> {faq.body.li2} </li>
+              </ul>
+            </p>
+            <p>
+              {faq.body.p9_1}
+              <b> {faq.body.p9_b} </b>
+              {faq.body.p9_2}
+              <ul>
+                <li> {faq.body.li3} </li>
+                <li> {faq.body.li4} </li>
+              </ul>
+            </p>
+            <p> {faq.body.p10} </p>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+/*
+--SECTIONS OBJECT DOC--
+
+Format:
+  section
+  |    title
+  |    body
+  |    |  enumerated HTML content elements
+
+  anchor tag (object)
+  |   text
+  |   link
+
+Naming schema for HTML content elements:
+For a series of elements in the body, each tag type is enumerated as follows
+-Example:
+  <p> ..1.. </p>
+  <p> ..2.. </p>
+  <a> ..3.. </a>
+
+  Turns into...
+
+  body: {
+    p1: "..1..",
+    p2: "..2..",
+    a1: "..3.."
+  }
+
+In order to accommodate in-element modifications (bolding, linking, etc),
+each element with inline mods is split up into respective parts
+-Example:
+  <p> The quick brown <b>fox</b> jumps...</p>
+
+  Turns into...
+
+  body: {
+    p1_1: "The quick brown ",
+    p1_b: "fox",
+    p1_2: " jumps..."
+  }
+
+  This is then encoded into the HTML through the JS injection with React.
+*/
+
+const SECTIONS = {
```

Comment:
Perhaps extract this out to another file, along with the documentation comment? It'll likely make it easier to read through, and keep things more modularized.

Reply:
Resolved, moved policies to static folder in data/